bftengine/tests/simpleTest/CMakeLists.txt

```diff
@@ -1,26 +1,39 @@
 set(simpleTest_client_source_files
     client.cpp
-    config.cpp
+        test_comm_config.cpp
```

Comment:
nit: please follow the existing indentation style

Reply:
NP, missed it.